### PR TITLE
Updated to Polkadot-v1.1.0

### PR DESCRIPTION
**Pull request by ayushmishra2005**

### CODE REVIEW

**Review by al3mart — DISMISSED** [review]
LGTM

**Review by stiiifff — DISMISSED** [review]

**Comment by stiiifff** [comment]
Looks good, thanks @ayushmishra2005 !
Let's hold off a bit before merging though.
We want to release a new version (and perform a runtime upgrade), before merging the Polkadot v1.1.0 upgrade.

**Comment by hbulgarini** [comment]
when this is merged, do not forget to restore the branch  so we have specific release branches.

**Review by stiiifff — APPROVED** [review]

**Pull request merged**
